test/ServerlessClamscan.test.ts

```diff
@@ -155,6 +160,41 @@ test('expect VirusDefsBucket to use provided logs bucket', () => {
       LogFilePrefix: 'test',
     },
   });
+
+  const stack3 = new Stack();
+  const imported_logs_bucket = Bucket.fromBucketArn(stack3, 'rImportedLogsBucket', Fn.importValue('rLogsBucketArn'));
+  new ServerlessClamscan(stack3, 'default', {
+    defsBucketAccessLogsConfig: { logsBucket: imported_logs_bucket },
+  });
+  expect(stack3).toHaveResourceLike('AWS::S3::Bucket', {
+    LoggingConfiguration: {
+      DestinationBucketName:
+      {
+        'Fn::Select': [
+          0,
+          {
+            'Fn::Split': [
+              '/',
+              {
+                'Fn::Select': [
+                  5,
+                  {
+                    'Fn::Split': [
+                      ':',
+                      {
+                        'Fn::ImportValue': stringLike('*rLogsBucket*'),
+                      },
+                    ],
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      },
+      LogFilePrefix: ABSENT,
+    },
+  });
```

Comment:
```suggestion
  const stack3 = new Stack();
  new ServerlessClamscan(stack3, 'default', {
    defsBucketAccessLogsConfig: {
      logsBucket: Bucket.fromBucketName(stack3, 'rImportedLogsBucket', 'imported'),
    },
  });
  expect(stack3).toHaveResourceLike('AWS::S3::Bucket', {
    LoggingConfiguration: {
      DestinationBucketName: 'imported',
    },
  });
```

This can be written more succinctly

Reply:
Updated